### ISSUE TITLE
263 projected publication date

### ISSUE THREAD

**Issue by CECSpecialistI** [uwlib-cams/MARC2RDA#128]
https://github.com/uwlib-cams/MARC2RDA/blob/main/Working%20Documents/2XX.csv

**Comment by cwarc** [comment]
https://docs.google.com/spreadsheets/d/1LG03vnz7YnbN1A1Kum-3DLKry_QzWwzH80EyVTldwFc/edit#gid=1155606979

Hello Everyone,

Please see the first pass for field code 263 - Projected Publication Date (NR). Would anyone like to review it?

Thanks,
Alice

**Comment by cspayne** [comment]
@CECSpecialistI @sarahruska let's reassign this mapping

**Comment by tmqdeborah** [comment]
I recommend mapping 263$a to Note on manifestation http://rdaregistry.info/Elements/m/P30137 as an unstructured description with a prefix "Projected date of publication" (RDA-speak) or "Projected publication date" (MARC-speak)

**Comment by CECSpecialistI** [comment]
This is an easy mapping for practice. Sara, do you mind taking this one?

**Comment by sarahruska** [comment]
> I recommend mapping 263$a to Note on manifestation http://rdaregistry.info/Elements/m/P30137 as an unstructured description with a prefix "Projected date of publication" (RDA-speak) or "Projected publication date" (MARC-speak)

@tmqdeborah - what do you think about "has manifestation publication statement" with a prefix? http://www.rdaregistry.info/Elements/m/#P30289

I think it could function similarly to a note on manifestation, but offers more specificity about the details of the note? "Relates a manifestation to a manifestation statement about the publication, release, or issuing of a manifestation."

**Comment by CECSpecialistI** [comment]
Manifestation statements need to be transcribed, is my understanding. Since this is projected, it definitely doesn't appear on the manifestation. I think it needs to be a note.

**Comment by tmqdeborah** [comment]
I agree with @sarahruska and @CECSpecialistI that this field should be mapped as a _note on manifestation_ with a prefix: 'Projected date of publication:'

**Comment by sarahruska** [comment]
Thank you, both!
